src/frontend/middleware.go

```diff
@@ -35,6 +38,11 @@ type logHandler struct {
 	next http.Handler
 }
 
+var (
+	meter                 = global.MeterProvider().Meter("frontend")
+	httpRequestCounter, _ = meter.SyncInt64().Counter("http.server.request_count")
```

Comment:
Consider duration histogram which is more interesting (and if folks only want the count, they can configure the SDK) https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#http-server.

Reply:
I add `http.server.duration` metric, but I think this version is literally simple one, much more optimization can be added in the successive prs, for examples,

- import net/http related instrumentation library. https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation/net/http/otelhttp
- add more app related attributes

Reply:
here is a screenshot of raw bucket metric in prometheus, `http_server_duration_bucket`

http://127.0.0.1:9090/graph?g0.expr=http_server_duration_bucket&g0.tab=1&g0.stacked=0&g0.show_exemplars=0&g0.range_input=1h

![image](https://user-images.githubusercontent.com/11855957/174724286-f1f438ca-c310-498c-8fc2-698e99e8db16.png)